Applications/DataExplorer/DataView/DiagramView/DetailWindow.cpp

```diff
@@ -124,9 +124,8 @@ void DetailWindow::resizeWindow()
 
 void DetailWindow::addList(DiagramList* list)
 {
-	GeoLib::Color* c = GeoLib::getRandomColor();
-	QColor colour((*c)[0], (*c)[1], (*c)[2]);
-	delete c;
+	GeoLib::Color const c = GeoLib::getRandomColor();
```

Comment:
Above line does not copy. I cannot tell the respective line of the C++ standard, but I am very sure, that
```
GeoLib::Color const c = GeoLib::getRandomColor();
```
and
```
GeoLib::Color const c(GeoLib::getRandomColor());
```
are equivalent. Though, I think the variant with `=` is more readable (albeit the constructor call is somewhat obfuscated).